### PR TITLE
VRFv2 smoke test

### PR DESCRIPTION
**Pull request by skudasov**

### CODE REVIEW

**Review by gheorghestrimtu — DISMISSED** [review]

File: integration-tests/smoke/vrfv2_test.go [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+			cd, err = contracts.NewContractDeployer(nets.Default)
+			Expect(err).ShouldNot(HaveOccurred())
+			cls, err = client.ConnectChainlinkNodes(e)
+			Expect(err).ShouldNot(HaveOccurred())
+			nets.Default.ParallelTransactions(true)
+		})
+		By("Funding Chainlink nodes", func() {
+			err = actions.FundChainlinkNodes(cls, nets.Default, big.NewFloat(3))
+			Expect(err).ShouldNot(HaveOccurred())
+			// Edge case where simulated networks need some funds at the 0x0 address in order for keeper reads to work
```
Comment (jleeh) [inline integration-tests/smoke/vrfv2_test.go:69]: Is this needed for a VRFv2 test?

**Review by jleeh — CHANGES_REQUESTED** [review]

File: integration-tests/smoke/vrfv2_test.go (hunk 1 above)
Comment (skudasov) [reply integration-tests/smoke/vrfv2_test.go:69]: Yes, `0x0` must be funded otherwise `simulate` part of a pipeline will fail

Comment (jleeh) [reply integration-tests/smoke/vrfv2_test.go:69]: Just update the comment then to reflect that

**Review by jleeh — CHANGES_REQUESTED** [review]

File: integration-tests/smoke/vrfv2_test.go (hunk 1 above)
Comment (jleeh) [reply integration-tests/smoke/vrfv2_test.go:69]: I'd also add this as a constant so we're not copying this string in tests

**Review by jleeh — APPROVED** [review]

**Comment by makramkd** [comment]
Can you explain why yarn.lock is updated? Also, can you link to the PR in the integrations framework repo that these changes depend on?

**Review by makramkd — DISMISSED** [review]

**Review by jmank88 — APPROVED** [review]

**Pull request merged**